templates/cli/scoop/appwrite.json.twig

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://raw.githubusercontent.com/ScoopInstaller/Scoop/master/schema.json",
+    "version": "{{ sdk.version }}",
+    "description": "Appwrite cli project manager",
+    "license": "BSD-3-Clause",
+    "url": "https://github.com/{{ sdk.gitUserName }}/{{ sdk.gitRepoName | caseDash }}/releases/download/{{ sdk.version }}/{{ language.params.executableName }}-cli-win-x64.exe",
+    "bin": [
+        [
+            "{{ language.params.executableName }}-cli-win-x64.exe",
+            "{{ language.params.executableName|caseLower }}"
+        ]
+    ],
+    "homepage": "https://github.com/{{ sdk.gitUserName|url_encode }}/{{ sdk.gitRepoName|url_encode }}",
+    "checkver": {
+        "url": "https://api.github.com/repos/appwrite/sdk-for-cli/releases",
+        "jsonpath": "$[0].tag_name"
+    }

```

Comment:
Since the sdk-for-cli repository now does its releases through actual releases rather than Pre-Releases this can be simplified to just point at the git repository. this removes the burden of maintaining this configuration & keeping it up to date with the Github API.